core/coreobjects/include/coreobjects/callable_info_impl.h

```diff
@@ -27,10 +27,11 @@ BEGIN_NAMESPACE_OPENDAQ
 class CallableInfoImpl : public GenericStructImpl<ICallableInfo, IStruct>
```

Comment:
This is a small-ish issue, as we're redefining a struct type here. I'm not 100% sure if these types are ever added to the type manager (I don't think they are), so it's not too big a deal. 

Notably, the struct type for CallableInfo must be updated to include isConst.